client/src/modules/employees/registry/registry.js

```diff
@@ -42,124 +46,136 @@ function EmployeeRegistryController(
       headerCellFilter     : 'translate',
       footerCellClass      : 'text-center',
       sortingAlgorithm : Sorting.algorithms.sortByReference,
+      cellClass : muteDisabledCells,
     },
     {
       field                : 'code',
       displayName          : 'TABLE.COLUMNS.REGISTRATION_NUMBER',
       headerCellFilter     : 'translate',
       footerCellClass      : 'text-center',
+      cellClass            : muteDisabledCells,
     },
     {
       field            : 'display_name',
       displayName      : 'TABLE.COLUMNS.NAME',
       headerCellFilter : 'translate',
+      cellTemplate     : '/modules/employees/templates/employee.cell.html',
```

Comment:
You should add the same muted class to this cell if the employee is disabled.

Reply:
Can you make these grey too?

![image](https://user-images.githubusercontent.com/896472/151404324-7c161233-2e31-4fbe-a86a-103f3f6f2918.png)